resolwe/flow/filters.py

```diff
@@ -616,7 +617,7 @@ class AnnotationValueMetaclass(ResolweFilterMetaclass):
     def __new__(mcs, name, bases, namespace):
         """Inject extensions into the filter."""
 
-        def filter_permissions(self, qs, value, original_filter):
+        def filter_permissions(self, qs, value: str, original_filter):
```

Comment:
Add type hint to the queryset. You could also type `original_filter`: it is a callable that accepts a queryset and a value and returns a new queryset.